ui-v2/src/components/events/event-details-page/event-details-display.tsx

```diff
@@ -0,0 +1,115 @@
+import { format } from "date-fns";
+import type { Event } from "@/api/events";
+import { Badge } from "@/components/ui/badge";
+import { Icon } from "@/components/ui/icons";
+import { EventResourceDisplay } from "../event-resource-display";
+import {
+	parseResourceType,
+	RESOURCE_ICONS,
+	RESOURCE_TYPE_LABELS,
+} from "../event-resource-display/resource-types";
+import { formatEventLabel } from "../events-timeline";
+
+type EventDetailsDisplayProps = {
+	event: Event;
+};
+
+type KeyValueProps = {
+	label: string;
+	children: React.ReactNode;
+};
+
+function KeyValue({ label, children }: KeyValueProps) {
+	return (
+		<div className="flex flex-col gap-0.5">
+			<span className="text-sm font-medium">{label}</span>
+			<div className="text-sm">{children}</div>
+		</div>
+	);
+}
```

Comment:
Can you follow a similar pattern as `ui-v2/src/components/deployments/deployment-metadata.tsx` and use `dd` and `dl` tags?